reporter/reporter.py

```diff
@@ -115,10 +121,81 @@ def notify():
     with CrateTranslator(DB_HOST, DB_PORT, DB_NAME) as trans:
         trans.insert([payload])
 
-    msg = "Notification successfully processed"
+    msg = 'Notification successfully processed'
     logger.info(msg)
     return msg
 
 
-if __name__ == '__main__':
-    app.run(host=LOCAL, port=8668)
+def query_1T1E1A(attrName, entityId, type=None):
+    """
+    See /entities/{entityId}/attrs/{attrName} in API Specification
+    quantumleap.yml
+    """
+    DB_HOST = os.environ.get('CRATE_HOST', 'crate')
+    DB_PORT = 4200
+    DB_NAME = "ngsi-tsdb"
+
+    from translators.crate import CrateTranslator
+    TIME_INDEX = CrateTranslator.TIME_INDEX_NAME
+    with CrateTranslator(DB_HOST, DB_PORT, DB_NAME) as trans:
+        attr_names = [TIME_INDEX, attrName]
+        vals = trans.query(attr_names=attr_names, entity_type=type,
+                           entity_id=entityId)
+    res = {
+        'data': {
```

Comment:
do we need the data wrapper?

Reply:
I thought this would be useful because the response will in many cases come with some metadata (eg, page number), so it's a good way to keep the pure data clean. It's analogous to the idea of "contextElement" in [STH](http://fiware-sth-comet.readthedocs.io/en/latest/raw-data-retrieval/index.html)

Reply:
in fact we can get some inspiration from things like [these](http://jsonapi.org/) and harmonise all responses.